src/arm/xnat.json

```diff
@@ -249,7 +250,7 @@
                             "resources": {
                                 "requests": {
                                     "cpu": 4,
-                                    "memoryInGb": 16
+                                    "memoryInGB": 16
```

Comment:
Is changing the capitalization correct? my concern is that the system is capitalization-sensitive and this may not be recognized and hence triggering an error.   If it not capitalization-sensitive then all good :)

Reply:
yes this is how it's defined in the JSON schema: https://schema.management.azure.com/schemas/2018-04-01/Microsoft.ContainerInstance.json